deploy/examples/create-external-cluster-resources.py

```diff
@@ -970,10 +966,10 @@ def get_rgw_fsid(self):
         if self._arg_parser.rgw_skip_tls:
             verify = False
         base_url =  self.endpoint_dial(rgw_endpoint,cert=cert) + "://"
-        base_url = base_url + rgw_endpoint + "/admin/info?"   
+        base_url = base_url + rgw_endpoint + "/admin/info?"
         params = {'format': 'json'}
         request_url = base_url + urllib.parse.urlencode(params)
-            
+
```

Comment:
@subhamkrai can we have a linter bot for this?

Reply:
right,  Ill take a look will open issue  to track